### ISSUE TITLE
Not matching Darcula; not highlighting all objects

### ISSUE THREAD

**Issue by tir38** [doums/darcula#23]
I put `darcula.vim` in `~/.vim/colors`. When selecting this colorscheme (with `:colorscheme`) I don't see the expected colors. I sometimes dont even see objects colorized at all. I'm sure the syntax "discovery" is working right. If I use another color scheme (e.g. deus) everything works fine:

_java with deus_:
<img width="575" alt="Screen Shot 2021-07-13 at 3 36 46 PM" src="https://user-images.githubusercontent.com/1980553/125514639-9b1f914e-2a2f-4778-a6a0-b461bae88bc7.png">

_java with darcula_:

<img width="565" alt="Screen Shot 2021-07-13 at 3 36 32 PM" src="https://user-images.githubusercontent.com/1980553/125514674-23d6b93e-9d18-42f4-b479-9fe1c1f207d7.png">

_kotlin with deus_:

<img width="514" alt="Screen Shot 2021-07-13 at 3 33 09 PM" src="https://user-images.githubusercontent.com/1980553/125515209-a7c5f065-2e25-4152-a162-1e46e5c7ba98.png">

_kotlin with darcula_:
<img width="432" alt="Screen Shot 2021-07-13 at 3 33 49 PM" src="https://user-images.githubusercontent.com/1980553/125515173-c65011b8-6c27-4340-86f5-0a47320e9a59.png">

_kotlin inside IntelliJ_:

<img width="677" alt="Screen Shot 2021-07-13 at 3 34 18 PM" src="https://user-images.githubusercontent.com/1980553/125515226-c3b1579e-222d-4188-bb9b-280ff8b80bb4.png">

**Comment by asousajnri** [comment]
For me I don't see the expected colors also.

![Screen Shot 2022-02-11 at 00 37 16](https://user-images.githubusercontent.com/51304001/153533970-21aa7c37-8479-4ce3-b3f3-9f0fe7b541c2.png)

**Comment by randy** [comment]
> For me I don't see the expected colors also.
> 
> ![Screen Shot 2022-02-11 at 00 37 16](https://user-images.githubusercontent.com/51304001/153533970-21aa7c37-8479-4ce3-b3f3-9f0fe7b541c2.png)

@asousajnri, I have the config as being `syntax on`, not `syntax enable`; maybe both work? I also don't use lightline, so maybe there's an incompatibility beyond what's mentioned in the readme..
 
For me, the problem is that comments appear to be highlighted. Language doesn't seem to matter. It's happening in Java, YAML, the .vimrc file, and bash scripts.
Here's a YAML:
![image](https://user-images.githubusercontent.com/22246488/190858966-1a46025a-7caa-4a2c-937a-226160501f95.png)

A bash script (yes, I use zsh)
![image](https://user-images.githubusercontent.com/22246488/190859261-d46057a6-7f5f-4af5-8763-cc3dff1faf65.png)

I've been poking around looking for the config for this and will probably eventually find it, but if anyone can point me in the right direction, I'd appreciate it! Also, if it's just a bug, then I'd be happy to start a new issue to keep things organized.

**Comment by garrofederico** [comment]
I'm having exactly the same issues with commented lines in yaml and also the colors are not correct. 
<img width="1420" alt="image" src="https://user-images.githubusercontent.com/68641690/191993567-e879ad7c-27c0-4ac2-b296-8f0d037742c2.png">
For reference, I have [Upgraded my macOS to Monterrey 12.5.1](https://stackoverflow.com/questions/73802870/upgraded-my-macos-to-monterrey-12-5-1-and-now-the-vim-colors-are-different-how) and ever since, some colorschemes look different than before.
@randy , Did you find a solution to this btw?

**Comment by randy** [comment]
@garrofederico, no, I sort of gave up and am living with it for now. Maybe in my non-existent free time I'll figure it out one day! 😄

**Comment by mammothbane** [comment]
Same issue &mdash; pretty sure at least in my case this is because I switched to treesitter, which darcula doesn't support currently

**Comment by mammothbane** [comment]
Oh, nevermind, it does, maybe my darcula is out of date.
